ibr/png-parser/REDAME.md

```diff
@@ -0,0 +1,16 @@
+## Instructions for Running the PNG Parser:
+
+1. Compile the proto definition to python classes:
+	- Install latest release of protocol buffers
+	    - go to [https://developers.google.com/protocol-buffers/docs/downloads]()
+	    - click on latest version release page
+	    - click on protobuf-$VERSION-$PLATFORM.zip to download the protocol buffers pre-built binary
+	    - unzip downloaded file
+	    - place bin/protoc somewhere in your $PATH
+	- move to current directory `cd digitalbuildings/ibr/png-parser`
+	- compile the proto definition: `protoc -I=../ibr-sdk/proto/ --python_out=. ibr.proto`
```

Comment:
this is where the python proto is being created, but im not seeing where ibr_pb2 is explicitly defined and imported into the python module? could we be more explicit with this?

Reply:
ibr_pb2 is the default output name when compiling ibr.proto. It is imported into the create_ibr.py module. I'll add a note specifying the default output name of this command.